### ISSUE TITLE
Dynamic lights stutter a lot vs Magnesium/Rubidium Dynamic Lights

### ISSUE THREAD

**Issue by ghost** [TexBlock/RubidiumZoomer#3]
Rubidium Toolkit:

https://user-images.githubusercontent.com/63958709/198874318-6c51f0f3-29e4-4e09-94b7-b23406dc950c.mp4

Magnesium/Rubidium Dynamic Lights + Magnesium/Rubidium Extras:

https://user-images.githubusercontent.com/63958709/198874373-0d0c38e5-756e-4a12-8f6d-2d651b6741a3.mp4

(Sorry for the poor video quality, I had to compress them to 10MB)

Other mods used:

![mods](https://user-images.githubusercontent.com/63958709/198874440-74e4b5e1-56f4-423b-918b-8312b5ef3d54.png)

Both instances were run with the same JVM arguments and the same Java version.

**Comment by TexBlock** [comment]
Rubidium Toolkit version？

**Comment by ghost** [comment]
Latest (1.0.3)

**Comment by TexBlock** [comment]
I think some feature of Magnesium/Rubidium Extras is optimized for dynamic light sources, or should I roll back some features that I removed?

**Comment by TexBlock** [comment]
Fix in [mc1.16.5-v1.0.4](https://github.com/TexTrueStudio/RubidiumToolkit/releases/tag/mc1.16.5-v1.0.4)

**Comment by Redls07** [comment]
Not fixed

https://user-images.githubusercontent.com/88981481/216054086-c08a16a7-bcad-4320-92ea-81596247ff1b.mp4

**Comment by ghost** [comment]
Yup, same here, the issue is still present. Magnesium/Rubidium DL works fine, Rubidium Toolkit stutters/lags a lot.

**Comment by Kasualix** [comment]
😨

**Comment by Kasualix** [comment]
May you test it with [this fork](https://github.com/MCTeamPotato/Rubidium-Extra-Potato-Edition/releases/download/1.3.6/magnesium_extras-mc1.16.5_v1.3.6.jar) please?
